pkg/jsonhttp/jsonhttptest/jsonhttptest.go

```diff
@@ -137,6 +137,21 @@ func ResponseDirectSendHeadersAndReceiveHeaders(t *testing.T, client *http.Clien
 	return resp.Header
 }
 
+func ResponseDirectSendHeadersAndDontCheckResponse(t *testing.T, client *http.Client, method, url string, body io.Reader, responseCode int, headers http.Header) (http.Header, []byte) {
```

Comment:
maybe a comment would be nice, although this mouthful of a name is almost self commenting

Reply:
:-)  Added some comments